pkg/innerring/processors/container/processor.go

```diff
@@ -273,6 +274,11 @@ func (cp *Processor) AddContainerStructs(ctx context.Context) error {
 	for ; ; time.Sleep(5 * time.Second) {
 		txRes, err := fsChain.CallNotary(ctx, cnrContract, fschaincontracts.AddContainerStructsMethod)
 		if err != nil {
+			if errors.Is(err, client.ErrNonAlphabet) {
```

Comment:
But you can check `cp.alphabetState.IsAlphabet()` above without any changes to `morph/client`.